CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+# 19.0.0
+
+* Remove default cache. This was previously set to `ActiveSupport::Cache::NullStore.new` which is now deprecated. If you are using this gem in a Rails app we continue to use the value of `Rails.cache` as before.
```

Comment:
It's not actually `ActiveSupport::Cache::NullStore` that's deprecated. Rather it's the [6.1 `Cache.format_version` that's deprecated](https://github.com/rails/rails/blob/36c1591bcb5e0ee3084759c7f42a706fe5bb7ca7/activesupport/lib/active_support/cache.rb#L766). The reason we're seeing the deprecation warning is because [the default cache format version remains at 6.1 even in ActiveSupport 7.1](https://github.com/rails/rails/blob/c0be0a2534837ed15a19135c1f5d9238206c5595/activesupport/lib/active_support/cache.rb#L55) and we're not setting it to something greater before initializing the `NullStore`.

Reply:
Ah, I see. I guess the reason for this is to allow you to choose when to invalidate your cache and upgrade to the new format version?